src/gpu_sysman_test.c

```diff
@@ -68,6 +68,25 @@
 #define FP_LAYOUT_NEED_NOTHING 1
 #include "gpu_sysman.c" /* test this */
 
+/* utilities needed from collectd core */
+#include <unistd.h>
+/* common.c fails to -Werror=unused-parameter, so
+ * string helpers still need mocked versions.
+ */
+#if 0
+#include <dirent.h>
+#include <math.h>
+#include <sys/stat.h>
+#include <sys/types.h>
+/* comment to avoid contrib/format.sh breaking code */
+#include "utils/common/common.c"
+#endif
```

Comment:
Is this left-over from development or the preferred solution should we be able to resolve the issues with `common.c`?

Reply:
I added commit fixing includes in `common.c` & `strbuf.c`, and removed these from the test code.